<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Transfer and Patching Management</string>
  </property>
  <widget class="QTextBrowser" name="sourceFile">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>241</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>1</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Single Server</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>34</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Remote Server IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostIP">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>39</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostUsr">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>363</x>
       <y>42</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>119</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>151</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>101</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 0);</string>
     </property>
     <property name="text">
      <string>eg. /opt/tomcat/webapps/......</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkPort">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>75</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>default Port</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>73</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Remote Server Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="hostPass">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>73</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>8</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="destPath">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>119</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>/opt/tomcat/webapps/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel">
     <property name="geometry">
      <rect>
       <x>213</x>
       <y>151</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkUsr">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>43</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>default User</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>151</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseFile">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>8</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose File...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vmPort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>363</x>
       <y>74</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Multiple Server</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>47</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Remote Server IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>52</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>128</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitM">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>157</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>110</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 0);</string>
     </property>
     <property name="text">
      <string>eg. /opt/tomcat/webapps/......</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>82</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Remote Server Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>82</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>6</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Select File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>128</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>/opt/tomcat/webapps/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelM">
     <property name="geometry">
      <rect>
       <x>214</x>
       <y>157</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendM">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>157</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>6</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose File...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>147</x>
       <y>33</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 85, 0);</string>
     </property>
     <property name="text">
      <string>Use Semicolon (;) as a seperator</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>217</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>396</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination Files</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="destFile">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>420</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>chooseFile</tabstop>
  <tabstop>hostIP</tabstop>
  <tabstop>checkUsr</tabstop>
  <tabstop>hostUsr</tabstop>
  <tabstop>hostPass</tabstop>
  <tabstop>checkPort</tabstop>
  <tabstop>vmPort</tabstop>
  <tabstop>destPath</tabstop>
  <tabstop>send</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>exit</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>sendM</tabstop>
  <tabstop>cancelM</tabstop>
  <tabstop>exitM</tabstop>
  <tabstop>sourceFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
